clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -722,7 +724,83 @@ describe("Fides-js TCF", () => {
           ).to.eql(true);
         });
       });
-
```

Comment:
nit: I really don't like the lack of newlines in between specs, is that something your IDE is doing? 😆

Reply:
oh, this was a rebase thing I probably should've kept. Will update!